deploy/helm/metacontroller/templates/statefulset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "metacontroller.fullname" . }}
+  namespace: {{ include "metacontroller.namespace" . }}
+  labels:
+    {{- include "metacontroller.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  serviceName: ''
+  selector:
+    matchLabels:
+      {{- include "metacontroller.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "metacontroller.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "metacontroller.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+          - /usr/bin/metacontroller
+          args:
+          - --logtostderr
```

Comment:
Those args should be configurable and allow to set any of the metacontroller args. I recommend to do some templating there and define default set in `values`